src/services/LegacyDomainService.ts

```diff
@@ -0,0 +1,177 @@
+import { Context, HttpRequest } from "@azure/functions";
```

Comment:
small nit: lower case "L" in file naming for consistency

Reply:
renamed